app/src/main/java/io/gnosis/safe/ui/dialogs/EnsInputDialog.kt

```diff
@@ -57,6 +58,8 @@ class EnsInputDialog : BaseViewBindingDialogFragment<DialogEnsInputBinding>() {
             backButton.setOnClickListener { dismiss() }
             confirmButton.setOnClickListener { onClick.offer(Unit) }
             dialogEnsInputUrl.showKeyboardForView()
+            //TODO: Replace with network name selected for safe addition (& set colors accordingly)
+            chainRibbon.text = BuildConfig.BLOCKCHAIN_NAME
```

Comment:
we should have chain data here that was passed in a bundle from chain selection screen